ee/localserver/server.go

```diff
@@ -119,6 +119,7 @@ func New(ctx context.Context, k types.Knapsack, presenceDetector presenceDetecto
 	mux.Handle("/zta", ls.requestDt4aInfoHandler())
 
 	mux.Handle("/v3/dt4a", dt4aAuthMiddleware.Wrap(ls.requestDt4aInfoHandler()))
+	mux.Handle("/v3/accelerate", dt4aAuthMiddleware.Wrap(ls.requestDt4aAccelerationHandler()))
```

Comment:
```suggestion
	mux.Handle("/zta", ls.requestDt4aInfoHandler())
	mux.Handle("/accelerate", ls.requestDt4aInfoHandler())
	
	mux.Handle("/v3/dt4a", dt4aAuthMiddleware.Wrap(ls.requestDt4aInfoHandler()))
	mux.Handle("/v3/accelerate", dt4aAuthMiddleware.Wrap(ls.requestDt4aAccelerationHandler()))
```

Maybe, could skip this?

Reply:
ahh woops, I didn't think about that .... @RebeccaMahany , I think the origin stuff cannot go into the dt4a auth middleware until we transition over to that completely

Reply:
I just removed the middleware when testing locally -- I figured we didn't need an unauthenticated accelerate endpoint that was more permanent than that

Reply:
We might not. Coordinating with other teams is always 🤷 

I'm good merging as is

Reply:
disregard my last comment, I was confused, it's still in there

Reply:
(They can use the regular /zta endpoint with the `OverlyBroad` flag set to do acceleration until they're ready to do auth for both /v3/dt4a and v3/accelerate)

Reply:
James and I chatted offline re: origin header -- I added the origin header checks to the dt4a auth middleware, but also left it in the handler hit by /zta directly, so we should be good there!